### DISCUSSION TITLE
[3.x] Add One in all command for nested modules

### DISCUSSION THREAD

**Discussion by haringsrob** [area17/twill#2439] [Ideas]
## Summary

When generating child-parent nested modules, a few manual actions are needed.

## Describe the solution you'd like

We could add `php artisan twill:make:nestedModule parent child`

## Additional context

This will require more code generation

**Comment by aksiome** [comment]
Hey @haringsrob,

I think it's best to just change the generation of the `make:module` command. We already have: `php artisan twill:make:module issueArticles --parentModel=Issue ` which should be enough information to generate the right controller. I even feel that the parentModel could be optional by just looking at the name of the module. Also maybe it's a good idea to change the generation to use the new setUpController method. I can try to work on this when I'll have a bit of time!
